shopinvader_api_sale_loyalty/models/sale_order.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    # TODO: needed?
+    reward_amount_tax_incl = fields.Float(compute="_compute_reward_total_tax_incl")
```

Comment:
In Odoo 14 the cart response returned the reward amount (tax excluded) + the reward amount tax included. In version 16 the reward_amount computation changed, it sometimes contains the amount tax included, sometimes tax excluded (depending on the type of tax on the related products).
I think returning the reward_amount (default Odoo field) is enough.